src/openPr.js

```diff
@@ -25,13 +29,18 @@ const addArtifact = async (inputs, releaseId) => {
   return artifact
 }
 
-const tryGetReleaseNotes = async (token, newVersion) => {
+const tryGetReleaseNotes = async (token, baseRelease, newVersion) => {
   try {
-    const latestRelease = await fetchLatestRelease(token)
-    if (!latestRelease) {
-      return
+    let baseVersion = baseRelease
+    if (!baseRelease) {
+      const latestRelease = await fetchLatestRelease(token)
+      if (!latestRelease) {
+        return
+      }
+      const { tag_name } = latestRelease
+      baseVersion = tag_name
```

Comment:
this logic is a bit confusing, let's try to write it in a more intuitive way or extract in a function